packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -65,9 +88,29 @@ export class GraphQLRequestClient implements GraphQLClient {
   private client: Client;
   private headers: Record<string, string> = {};
   private debug: Debugger;
-  private retries: number;
   private abortTimeout?: TimeoutPromise;
   private timeout?: number;
+  private retries: number;
+  private retryStrategy: RetryStrategy;
+  private defaultRetryStrategy: RetryStrategy = {
+    shouldRetry: (error: ClientError, attempt: number) => {
+      const retryableStatusCodes = [429, 502, 503, 504, 520, 521, 522, 523, 524];
```

Comment:
Is our final decision to include all the 5xx codes by default? This will happen for both SXP and XMC

Reply:
Yes we discussed that yesterday but will clarify more with Adam.